Jetty 11 build still depends on apache-jstl

**Jetty version**
11.0.1-SNAPSHOT

**Description**

```
[INFO] Jetty :: GCloud .................................... SUCCESS [  0.072 s]
[INFO] Jetty :: GCloud :: Session Manager ................. SUCCESS [  0.961 s]
[INFO] Jetty :: Memcached ................................. SUCCESS [  0.100 s]
[INFO] Jetty :: Memcached :: Sessions ..................... SUCCESS [  0.583 s]
[INFO] Jetty Tests :: Sessions :: Parent .................. SUCCESS [  0.081 s]
[INFO] Jetty Tests :: Sessions :: Common .................. SUCCESS [  1.463 s]
[INFO] Jetty :: NoSQL Session Managers .................... SUCCESS [  0.409 s]
[INFO] Jetty :: Test Keystore ............................. SUCCESS [  0.278 s]
[INFO] Jetty :: Home Assembly ............................. SUCCESS [ 10.394 s]
[INFO] Demo :: Embedded Jetty ............................. FAILURE [  7.296 s]
[INFO] Jetty :: Ant Plugin ................................ SKIPPED
[INFO] Jetty :: Websocket :: Core :: Tests ................ SKIPPED
...(snip)...
[INFO] Jetty :: Documentation :: AsciiDoctor Extensions ... SKIPPED
[INFO] Jetty :: Documentation ............................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:30 min
[INFO] Finished at: 2021-02-19T13:11:23-06:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project demos-jetty-embedded: Could not resolve dependencies for project
 org.eclipse.jetty.demos:demos-jetty-embedded:jar:11.0.1-SNAPSHOT: Could not find artifact
 org.eclipse.jetty:apache-jstl:jar:11.0.1-SNAPSHOT in oss-snapshots
 (https://oss.sonatype.org/content/repositories/snapshots) -> [Help 1]
```

--- a/demos/embedded/pom.xml
+++ b/demos/embedded/pom.xml
@@ -123,7 +123,7 @@
     </dependency>
     <dependency>
       <groupId>org.eclipse.jetty</groupId>
-      <artifactId>apache-jstl</artifactId>
+      <artifactId>glassfish-jstl</artifactId>
       <version>${project.version}</version>
     </dependency>
     <dependency>

--- a/javadoc/pom.xml
+++ b/javadoc/pom.xml
@@ -201,7 +201,7 @@
     </dependency>
     <dependency>
       <groupId>org.eclipse.jetty</groupId>
-      <artifactId>apache-jstl</artifactId>
+      <artifactId>glassfish-jstl</artifactId>
       <version>${project.version}</version>
       <scope>provided</scope>
     </dependency>

--- a/jetty-bom/pom.xml
+++ b/jetty-bom/pom.xml
@@ -57,7 +57,7 @@
       </dependency>
       <dependency>
         <groupId>org.eclipse.jetty</groupId>
-        <artifactId>apache-jstl</artifactId>
+        <artifactId>glassfish-jstl</artifactId>
         <version>11.0.1-SNAPSHOT</version>
       </dependency>
       <dependency>

--- a/jetty-jspc-maven-plugin/pom.xml
+++ b/jetty-jspc-maven-plugin/pom.xml
@@ -103,7 +103,7 @@
     </dependency>
     <dependency>
       <groupId>org.eclipse.jetty</groupId>
-      <artifactId>apache-jstl</artifactId>
+      <artifactId>glassfish-jstl</artifactId>
       <version>${project.version}</version>
     </dependency>
     <dependency>

--- a/jetty-maven-plugin/pom.xml
+++ b/jetty-maven-plugin/pom.xml
@@ -243,7 +243,7 @@
     </dependency>
     <dependency>
       <groupId>org.eclipse.jetty</groupId>
-      <artifactId>apache-jstl</artifactId>
+      <artifactId>glassfish-jstl</artifactId>
       <version>${project.version}</version>
     </dependency>
     <dependency>

